src/lib/keys/KeyManagerWorker.ts

```diff
@@ -38,9 +38,14 @@ const keyManagerWorker = {
           km = ds.get_key_manager()
         }
         if (km) {
-          resolve(km.get_pgp_fingerprint().toString('hex'));
+          if (km.get_pgp_fingerprint()) {
```

Comment:
You need to add more inline comments to express the "human" logic of what you are trying to do here. And also "WHY". The order is always: `Why` -> `How` -> `What.

Reply:
I've added some comments in to make it a little more clear why we verify the data only if there is a pgp finger print.